src/components/Table/Pagination.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { Pagination, Select } from 'antd';
+
+const PaginationComponent = ({ totalItems, pageSize, currentPage, onPageChange, onPageSizeChange }) => {
+  const handlePageSizeChange = (value) => {
+    onPageSizeChange(parseInt(value));
+  };
+
+  const handlePageChange = (page) => {
+    onPageChange(page);
+  };
+
+  const startIndex = (currentPage - 1) * pageSize;
+  const endIndex = startIndex + pageSize;
+
+  const paginationConfig = {
+    pageSize: pageSize,
+    total: totalItems,
+    current: currentPage,
+    onShowSizeChange: handlePageSizeChange,
+    onChange: handlePageChange,
+    showTotal: (total, range) => `${range[0]}-${range[1]} of ${total} Projects`,
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < Math.ceil(totalItems / pageSize)) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  const handleLastPage = () => {
+    onPageChange(Math.ceil(totalItems / pageSize));
+  };
+
+  const isLastPage = currentPage === Math.ceil(totalItems / pageSize);
+
+  const pageSizeSelect = (
+    <Select value={pageSize.toString()} onChange={handlePageSizeChange}>
+      {['5', '10', '20'].map((value) => (
```

Comment:
One more time you ignore my comment. 1. move map outside of jsx
2. move array in separate variable. 3. use 5 10 and 20 entries is small amount of data, use 10 20 50.